style.css

```diff
@@ -101,4 +101,27 @@ button {
     text-transform: uppercase;
     background: var(--accent-color);
     color: #fff;
+}
+
+
+/*
+**Desktop draft
+*/
+
+@media only screen and (min-width: 961px) {
```

Comment:
We don't really need `only screen` here.

Reply:
Why `961px` ? I mean why this size specifically?

Reply:
Really ? I'll find out why it's not mandatory in our case

Well, it's arbritary, i have this habit to use max width pixel value, is this a bad practice ?